production-offline-deployment-using-tiup.md

```diff
@@ -12,7 +12,7 @@ This document describes how to deploy a TiDB cluster offline using TiUP.
 
 ### Option 1: Download the official TiUP offline component package
 
-Download the prepared TiUP offline package from the [Download TiDB](https://pingcap.com/download) page.
+Download the offline mirror package of TiDB server (including the TiUP offline component package) from the [Download TiDB](https://pingcap.com/download-cn/) page.
```

Comment:
```suggestion
Download the offline mirror package of the TiDB server (including the TiUP offline component package) from the [Download TiDB](https://pingcap.com/download/) page.
```
For readers who speak English, give them the English version of the link. Be cautious of this kind of translation and use your critical thinking to get close to the truth! 😉 @ireneontheway